### PR TITLE
Add tests for recaptcha service #599

### PR DESCRIPTION
**Pull request by pcaston**
Submitting pull request for comment and assistance.
Final step in cucumber feature failing.
Google sends a message that browser is not supported (Xvfb)
Attempted to stub google recaptcha without success.

### CODE REVIEW

**Comment by pcaston** [comment]
Closed while investigating other failing tests.

**Comment by patmbolger** [comment]
@pcaston - Hi Paul .... I did not see this until today.  I can take a look when you reopen the PR.

**Comment by pcaston** [comment]
Re-openned to re-run tests

**Comment by pcaston** [comment]
The features/event_management.feature:65 test is also failing for me on the development branch.

File: app/services/cruncher_service.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class CruncherService
   require 'rest-client'
   require 'json'
 
-  @@auth_token = nil
+  @auth_token = nil
```
Comment (patmbolger) [inline app/services/cruncher_service.rb]: This change should be undone.  `@@auth_token` is a class variable, _not_ instance instance var.  In fact, this class is never instantiated as an instance (all the methods are class methods).

File: spec/support/service_stub_helpers.rb [hunk 2]
```diff
@@ -163,9 +163,9 @@ def stub_cruncher_job_create_retry_auth_fail
         .to_raise(RestClient::Unauthorized)
     end
 
-    def stub_cruncher_job_create_fail(result_code)
+    def stub_cruncher_job_create_fail(resultCode)
```
Comment (patmbolger) [inline spec/support/service_stub_helpers.rb]: I think hound wants us to use `result_code` (which is snake_case) rather than camel case.

**Comment by pcaston** [comment]
Re-open to rerun semaphore tests.

File: features/pages.feature [hunk 3]
```diff
@@ -12,7 +12,7 @@ Scenario: make contact and check the recaptcha
   And I fill in "Message" with "Hi Metplus"
   And I have checked the recaptcha
   And I click the "Send Message" button
-  And I wait 1 second
+  And I wait 2 seconds
```
Comment (patmbolger) [inline features/pages.feature]: @pcaston - waiting on this step will never work.  Please see comments in my PR that shows my tweaks to your feature branch: https://github.com/pcaston/MetPlus_PETS/pull/1

**Comment by patmbolger** [comment]
Looks good - merging.

**Pull request merged**